File: modules/version-consistency-checker/src/version_consistency_checker/utils/normalize_tool_name.py
def normalize_tool_name(key: str) -> str:
  """Normalize a mise tool key to a short tool name.

  Rules:
  - If the key contains ':', take the part after ':' as the package name.
  - If the package name contains '/', take the part after the last '/'.
  """
  package = key.split(":", 1)[-1]
  return package.rsplit("/", 1)[-1]

File: modules/version-consistency-checker/src/utils/normalizeToolName.ts
/**
 * Normalize a mise tool key to a short tool name.
 *
 * Rules:
 * - If the key contains ':', take the part after ':' as the package name.
 * - If the package name contains '/', take the part after the last '/'.
 */
export const normalizeToolName = (key: string): string => {
  const colonIndex = key.indexOf(":");
  const package_ = colonIndex !== -1 ? key.slice(colonIndex + 1) : key;
  return package_.split("/").at(-1) ?? "";
};
